### PR TITLE
srx duplicate policy check (for same from/to zone pair)

### PR DESCRIPTION
**Pull request by sjtarik**
…m-zone to-zone pair in the same policy file which would overwrite each other on srx platform

### CODE REVIEW

File: lib/junipersrx.py [hunk 1]
```diff
@@ -318,6 +322,7 @@ def _TranslatePolicy(self, pol, exp_info):
     Raises:
       UnsupportedFilterError: An unsupported filter was specified
       UnsupportedHeader: A header option exists that is not understood/usable
+      SRXDuplicatePolicy: A policy with multiple definition for same from-zone to-zone pair
```
Comment (ankenyr) [inline lib/junipersrx.py]: nit: A policy with duplicate from-zone to-zone pairs.

**Comment by ankenyr** [comment]
Going to need to roll this back, seems this checks for duplicate names across the boundaries which it should not. It is common to use include files to abstract certain terms that need to be used everwhere. Those include files would be used in multiple headers and that would trigger this error message.

**Comment by sjtarik** [comment]
@ankenyr, could you give an example of boundary condition? Or even better point to a unit test. Let me fix it since I need this logic to resolve conflicts in address book resolution?
